tools/test-static-code

```diff
@@ -80,4 +81,16 @@ else
     echo "ok 4 unsafe-security-policy"
 fi
 
+#
+# Bad paths or modifications in patternfly for fonts
+#
+
+if grep -R "\.\./fonts/OpenSans\|fonts/.*eot\|truetype" $distdir/dist/*/*.css >&2; then
```

Comment:
I wonder  if that is really right. We ship the css in the *source* dir, as part of `make dist`, not in the build dir. So I believe this should just check `dist/*/*.css`? The build directory only has the `*.css.gz` variants.

Reply:
I guess it needs to check both? During development the $builddir/dist/*/*.css exists for me. Can we do both?

Reply:
OK, force-pushed my FIXUP again to do both.